orchagent/copporch.cpp

```diff
@@ -212,6 +212,47 @@ bool CoppOrch::applyAttributesToTrapIds(sai_object_id_t trap_group_id,
             return false;
         }
         m_syncdTrapIds[trap_id] = trap_group_id;
+
+        auto hostTbl_entry = m_trapid_hostif_table_map.find(trap_id);
```

Comment:
This is HostTable creation. Can you please separate this from `applyAttributesToTrapIds` function?